og_access/og_access.module

```diff
@@ -0,0 +1,215 @@
+    return [];
+  }
+
+  // The group IDs, that in case access is granted, will be recorded.
+  $gids = [];
+
+  if (Og::isGroup('node', $node->getType()) &&
+      $node->hasField(OgAccess::OG_ACCESS_FIELD) &&
+      !empty($node->{OgAccess::OG_ACCESS_FIELD}) && $node->{OgAccess::OG_ACCESS_FIELD}->value) {
+    // Private group.
+    $gids['node'][] = $node->id();
+  }
+
+  if ($node->hasField(OgAccess::OG_ACCESS_CONTENT_FIELD) &&
+      !empty($node->get(OgAccess::OG_ACCESS_CONTENT_FIELD))) {
+    $content_access = $node->get(OgAccess::OG_ACCESS_CONTENT_FIELD)->value;
+  }
+  else {
+    $content_access = OgAccess::OG_ACCESS_PUBLIC;
+  }
+
+  switch ($content_access) {
+    case OgAccess::OG_ACCESS_PUBLIC:
+      // Skip non-group content nodes.
+      if (!Og::isGroupContent('node', $node->getType())) {
+        break;
+      }
+
+      $has_private = FALSE;
+      /** @var \Drupal\og\OgGroupAudienceHelper $audience_helper */
+      $audience_helper = \Drupal::service('og.group_audience_helper');
+      foreach ($audience_helper->getAllGroupAudienceFields('node', $node->getType()) as $field_name => $field) {
+        foreach ($node->get($field_name)->referencedEntities() as $group) {
+          $list_gids[$group->getEntityTypeId()][] = $group->id();
+
+          if ($has_private) {
+            // We already know we have a private group, so we can avoid
+            // re-checking it.
+            continue;
+          }
+
+          if ($group->hasField(OgAccess::OG_ACCESS_FIELD) && !empty($group->get(OgAccess::OG_ACCESS_FIELD)) &&
+              $group->get(OgAccess::OG_ACCESS_FIELD)->value) {
+            $has_private = TRUE;
+          }
+        }
+      }
+      if ($has_private) {
+        $gids = array_merge_recursive($gids, $list_gids);
+      }
+      break;
+
+    case OgAccess::OG_ACCESS_PRIVATE:
+      $list_gids = [];
+      /** @var \Drupal\og\OgGroupAudienceHelper $audience_helper */
+      $audience_helper = \Drupal::service('og.group_audience_helper');
+      foreach ($audience_helper->getAllGroupAudienceFields('node', $node->getType()) as $field_name => $field) {
+        foreach ($node->get($field_name)->referencedEntities() as $group) {
+          $list_gids[$group->getEntityTypeId()][] = $group->id();
+        }
+      }
+
+      $gids = array_merge_recursive($gids, $list_gids);
+      break;
+  }
+
+  foreach ($gids as $group_type => $values) {
+    foreach ($values as $gid) {
+      $grants[] = [
+        'realm' => OgAccess::OG_ACCESS_REALM . ':' . $group_type,
+        'gid' => $gid,
+        'grant_view' => 1,
+        'grant_update' => 0,
+        'grant_delete' => 0,
+      ];
+    }
+  }
+
+  return !empty($grants) ? $grants : [];
+}
+
+/**
+ * Implements hook_form_alter().
+ */
+function og_access_form_alter(array &$form, FormStateInterface $form_state, $form_id) {
+  if ($form_state->getFormObject() instanceof BundleEntityFormBase) {
+    (new OgAccessBundleFormAlter($form_state->getFormObject()->getEntity()))
```

Comment:
```suggestion
    (new OgAccessBundleFormAlter($form_state->getFormObject()->getEntity(), \Drupal::service('string_translation')))
```

ArgumentCountError: Too few arguments to function Drupal\og_access\OgAccessBundleFormAlter::__construct(), 1 passed in /web/modules/contrib/og/og_access/og_access.module on line 138 and exactly 2 expected in /web/modules/contrib/og/og_access/src/OgAccessBundleFormAlter.php on line 46